src/runtime/threads/detail/scheduled_thread_pool.cpp

```diff
@@ -38,6 +38,9 @@ template class HPX_EXPORT hpx::threads::detail::scheduled_thread_pool<
 template class HPX_EXPORT hpx::threads::detail::scheduled_thread_pool<
     hpx::threads::policies::local_priority_queue_scheduler<hpx::compat::mutex,
         hpx::threads::policies::lockfree_abp_fifo>>;
+template class HPX_EXPORT hpx::threads::detail::scheduled_thread_pool<
+    hpx::threads::policies::local_priority_queue_scheduler<hpx::compat::mutex,
+        hpx::threads::policies::lockfree_abp_lifo>>;
 #endif
```

Comment:
If you add this specialization then you probably should add a command line option for ir as well.

Reply:
Yes, you're right, will add that.

Reply:
OTOH, instead of adding more schedulers to HPX core, we should consider creating a better mechanism (using the thread-pools) allowing for all schedulers (except the default one) to live outside of HPX core. What do you think?

Reply:
Not thought about it too much, but yes, that would probably be a good thing. How far outside HPX core are you thinking? This probably goes hand in hand with having a nicer way to create thread pools in general.